src/Files.App/Views/LayoutModes/GridViewBrowser.xaml

```diff
@@ -247,6 +247,7 @@
 		<DataTemplate x:Name="TilesBrowserTemplate" x:DataType="filesystem:ListedItem">
```

Comment:
Can you do the same for the GridItem template? This will only help for the Tiles layout.

Reply:
Sorry, I don't even know how to start. I haven't done such a thing before.

Reply:
It's the same thing you did here, just on line 57

Reply:
It's done

Reply:
Did you push the change yet?